### DISCUSSION TITLE
Example of styling individual points

### DISCUSSION THREAD

**Discussion by scimone** [patrykandpatrick/vico#812] [Ideas]
### Question

Hello, I recently found out that the version [Vico 2.0.0-alpha.23](https://patrykandpatrick.com/vico/releases/2.0.0-alpha.23/) brought a lot of good updates. I've been studying the documentation, but it only provides an example of how to apply points of one common style to all entries of a line. Is it possible to also provide a minimum example of styling individual points, e.g. using different point colors depending on the y value? 
I have tried, but as a beginner I'm having a hard time understanding the explanations. Thank you for your work!

### Vico version(s)

2.0.0-alpha.23

### UI framework(s)

Jetpack Compose

**Comment by Gowsky** [comment]
Hello, @scimone. We'll look into adding such guidance to the wiki. In the meantime, we'll respond with more information on how to do this under #827.
